server.js

```diff
@@ -1,30 +1,148 @@
-import express from "express";
+const express = require("express");
+const listEndPoints = require("express-list-endpoints");
 import cors from "cors";
+import topBooks from "./data/100-trending-books.json";
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// import avocadoSalesData from "./data/avocado-sales.json";
-// import booksData from "./data/books.json";
-// import goldenGlobesData from "./data/golden-globes.json";
-// import netflixData from "./data/netflix-titles.json";
-// import topMusicData from "./data/top-music.json";
-
-// Defines the port the app will run on. Defaults to 8080, but can be overridden
-// when starting the server. Example command to overwrite PORT env variable value:
-// PORT=9000 npm start
+// Defines the port the app will run on.
 const port = process.env.PORT || 8080;
 const app = express();
 
 // Add middlewares to enable cors and json body parsing
 app.use(cors());
 app.use(express.json());
+//app.use(bodyParser.json()) Damen hade detta i sitt exempel??
 
-// Start defining your routes here
+// Endpoint to return documentation using Express List Endpoints
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  const documentation = {
+    endpoints: listEndPoints(app), 
+    additionalInfo: {
+      '/books': {
+        description: "Get a list of books with optional filtering", 
+        queryParams: {
+          rating: "Filter books based on requested mininum rating"
+        }, 
+        example: "https://books-toplist-100.onrender.com/books/?rating=4.7"
+      }, 
+      '/genre': {
+        description: "Get a list of books with optional filtering based on genre", 
+        queryParams: {
+          rating: "Filter books based on requested genre and mininum rating"
+        }, 
+        example: "https://books-toplist-100.onrender.com/genre/fantasy/?rating=4.9"
+      }
+    }
+  }
+  res.json(documentation)
+});
```

Comment:
This is great! I didn't do this, didn't even think about it, but I can certainly see why it is important to have it in place. I will definitely have this in mind moving forward. Good job!